api/swagger.yml

```diff
@@ -415,6 +415,55 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error500'
+    get:
+      tags:
+        - Coordinator
+      summary: Get transactions that are in the pool.
+      operationId: getPoolTxs
+      parameters:
+        - name: state
+          in: query
+          required: false
+          description: State of the transactions, e.g. "pend"
+          schema:
+            $ref: '#/components/schemas/PoolL2TransactionState'
```

Comment:
Please add the following filters:
- hezEthereumAddress
- BJJ
- type
- accountIndex

Reply:
Also add the paginated related filters:
- limit
- fromItem
- order

I recommend to read the description at the beggining of the [swagger doc](https://apidoc.hermez.network/) where there is an explanation on how does pagination work in this API (as well as other relevant info)